internal/scan/scan_test.go

```diff
@@ -0,0 +1,43 @@
+package scan
+
+import (
+	"context"
+	"github.com/openshift/check-payload/internal/types"
+	"testing"
+)
+
+// TestRunLocalScan tests the RunLocalScan function with mock bundles.
+// TODO add non-FIPS compliant compiled binary to BadBundle to make a fail test case
+// TODO add FIPS compliant compiled binary to GoodBundle to make a better pass test case
+func TestRunLocalScan(t *testing.T) {
+	// Define test cases
+	testCases := []struct {
+		name           string
+		bundlePath     string
+		expectedResult bool // true if scan should pass, false if it should fail
+	}{
+		{"GoodBundle", "../test/resources/bundle-1", true},
+		//{"BadBundle", "../test/resources/bundle-2", false},
+	}
+
+	// Iterate over test cases
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			// Setup context and config
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+
+			var cfg types.Config
+			cfg.SetupDefault() // Assuming a method to set up default config
```

Comment:
Add a NewDefaultConfig() function makes sense... Keep the SetupDefault (or perhaps rename it to SetupDefaults()) .

Reply:
yep, changed it to the former suggestion, better fit for what it does.